### PR TITLE
testsuite/tester: ignore newline changes when comparing generated nim files

### PR DESCRIPTION
**Pull request by tmm1**
makes test pass for me on OSX

### CODE REVIEW

**Comment by Araq** [comment]
Shouldn't the generated files use LF everywhere?

**Comment by tmm1** [comment]
Yes the generated files are using LF, but the results contain CRLF as of b57f503cc444bf7ede3ea8093c98b95d205c9130.

Should I remove the CR in the results instead then?

**Comment by tmm1** [comment]
Tests are green on travis now. I'm building against nim@devel and removed the "\r" from generated files.

File: testsuite/tester.nim [hunk 1]
```diff
@@ -16,6 +16,7 @@ proc test(t, cmd: string) =
   if strip(readFile(dir & "tests" / nimFile)) !=
      strip(readFile(dir & "results" / nimFile)):
     echo "FAILURE: files differ: ", nimFile
+    discard execShellCmd("diff -uNdr " & dir & "tests" / nimFile & " " & dir & "results" / nimFile)
```
Comment (Araq) [inline testsuite/tester.nim]: Meh, I don't even know if i have a diff command.

**Pull request merged**